src/CMakeLists.txt

```diff
@@ -324,7 +324,7 @@ add_subdirectory(json_spirit)
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/src/xxHash")
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/src/rapidjson/include")
 
-find_package(fmt 6.0.0 QUIET)
+find_package(fmt 7.1.3 QUIET)
```

Comment:
why?

Reply:
6.0 failed linking for some of the bins. The error seemed related to a bug that was worked around
in fmt 7.1.3. See the discussion in
https://github.com/fmtlib/fmt/issues/1753

Reply:
i'd suggest keep it as it is, unless we are using some new features or bug fixes which does not exist in 6.0. more importantly, quite a few developers are still using the GCC for compiling Ceph, AFAICT, and they don't have 7.1.3 prepackaged in the distro they are using: ubuntu focal ships 6.1.2, EPEL8 has 6.2.1.

if you really want to ensure that the clang users can be warned probably you could add something like
```cmake
if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND
    fmt_VERSION VERSION_LESS "7.1.3") 
  message(WARNING "fmt compiled with GCC does not link with Clang, see https://github.com/fmtlib/fmt/issues/1753")
endif()
```

please note, this issue does not impact the users who only use clang or GCC for compiling libfmt and the application using it. that's another reason why we should not enforce this version or bail out if the version is lower than 7.1.3 .